src/libraries/System.Data.Common/src/System/Data/Common/DbDataRecord.cs

```diff
@@ -69,6 +70,7 @@ protected virtual DbDataReader GetDbDataReader(int i)
         // ICustomTypeDescriptor
         //
 
+#nullable disable
```

Comment:
Ah, are you disabling nullability around interface implementations for interfaces that haven't been annotated yet?  If so, I'd suggest to just leave that up to whomever annotates the interfaces.

Reply:
Yeah, I was planning to do another pass once those things are annotated. We can do it either way.